app/services/send_eoc_deadline_reminder_email_to_candidate.rb

```diff
@@ -1,17 +1,25 @@
 class SendEocDeadlineReminderEmailToCandidate
```

Comment:
Can you add a comment above the class declaration saying End Of Cycle for LSPs

Reply:
I'm happy to do this, but I'm not sure exactly what you need. Is it literally just `# EndOfCycle` ?